docs/sources/mimir/migration-guide/migrate-from-cortex.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Migrate from Cortex to Grafana Mimir"
+menuTitle: "Migrate from Cortex"
+description: "Learn how to migrate your deployment of Cortex to Grafana Mimir to simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage."
+weight: 10
+---
+
+# Migrate from Cortex to Grafana Mimir
+
+This document guides an operator through the process of migrating a Jsonnet deployment of [Cortex](https://cortexmetrics.io/) to Grafana Mimir.
+This document includes an overview of the steps required for any environment, and specific instructions for [environments deployed with Jsonnet](#migrating-to-grafana-mimir-using-jsonnet).
+To migrate a Helm deployment of Cortex refer to [Migrate from Cortex](/docs/helm-charts/mimir-distributed/latest/migration-guides/migrate-from-cortex).
+
+Grafana Mimir 2.0.0 includes significant changes that simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage.
+
+The changes focus on making Grafana Mimir easier to run out of the box, including:
+
+- Removing configuration parameters that don't require tuning
+- Renaming some parameters so that they're more easily understood
+- Updating the default values of some existing parameters
+
+The `mimirtool` automates configuration conversion.
+It provides a simple migration by generating Mimir configuration from Cortex configuration.
+
+## Before you begin
+
+- Ensure that you are running either Cortex 1.10.X or Cortex 1.11.X.
+
+  If you are running an older version of Cortex, upgrade to [1.11.1](https://github.com/cortexproject/cortex/releases) before proceeding with the migration.
+
+- Ensure you have installed Cortex alerting and recording rules as well as Cortex dashboards.
+
+  The monitoring mixin has both alerting and recording rules to install in either Prometheus or Cortex as well as dashboards to install in Grafana.
+  To download a prebuilt ZIP file that contains the alerting and recording rules, refer to [Release Cortex-jsonnet 1.11.0](https://github.com/grafana/cortex-jsonnet/releases/download/1.11.0/cortex-mixin.zip).
+
+  To upload rules to the ruler using mimirtool, refer to [mimirtool rules]({{< relref "../operators-guide/tools/mimirtool.md" >}}).
+  To import the dashboards into Grafana, refer to [Import dashboard](/docs/grafana/latest/dashboards/export-import/#import-dashboard).
```

Comment:
The doc validator is failing here, but this is the same string as used in the Helm docs, so I'm not sure what to do. Ideas, @osg-grafana ?